docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+        python manage.py migrate &&
+        python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db

```

Comment:
The `depends_on` option ensures that the `db` service is started before the `cinema` service. However, it does not wait for the database to be ready. Consider using a script or tool to check the database readiness if necessary, although the `wait_for_db` command in your Django management command should handle this.